src/db/sqlite/lrsql/sqlite/record.clj

```diff
@@ -89,6 +89,19 @@
     (when-not (some? (query-xapi-statement-stored-exists tx))
       (alter-xapi-statement-add-stored! tx)
       (migrate-xapi-statement-stored-times! tx))
+    (when-not (some? (query-state-document-last-modified-is-timestamp tx))
+      (migrate-timestamps-state-01! tx)
+      (migrate-timestamps-state-02! tx)
+      (migrate-timestamps-state-03! tx)
+      (migrate-timestamps-state-04! tx)
+      (migrate-timestamps-agent-profile-01! tx)
+      (migrate-timestamps-agent-profile-02! tx)
+      (migrate-timestamps-agent-profile-03! tx)
+      (migrate-timestamps-agent-profile-04! tx)
+      (migrate-timestamps-activity-profile-01! tx)
+      (migrate-timestamps-activity-profile-02! tx)
+      (migrate-timestamps-activity-profile-03! tx)
+      (migrate-timestamps-activity-profile-04! tx))
```

Comment:
As you said this looks absolutely disgusting, but given that it's just a one-time migration and the whole thing is conceptually simple it's an okay workaround. Really wish I knew of a way to do multi-statement commands in SQLite...